target/linux/ipq806x/files-4.19/arch/arm/boot/dts/qcom-ipq8064-wpq864.dts

```diff
@@ -333,6 +333,8 @@
 	fixed-link {
```

Comment:
Would it be too much to know Why? are you adding those options? I can already see, that you've added them :-)

Reply:
@ynezz can't really find a reason for this change... As this was done only for some board i dropped it. (Think it was in testing by chunkeey)